core/src/main/java/org/elasticsearch/indices/memory/IndexingMemoryController.java

```diff
@@ -170,11 +170,15 @@ public void run() {
                     indexShard.markAsInactive();
                 } catch (EngineClosedException e) {
                     // ignore
+                    logger.debug("ignore EngineClosedException while marking shard [{}][{}] as inactive", indexShard.shardId().index().name(), indexShard.shardId().id());
```

Comment:
move these to TRACE? they don'e really signal any issue but people may thing so...